test/cholesky_dense.cpp

```diff
@@ -0,0 +1,114 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <gtest/gtest.h>
+#include <arrayfire.h>
+#include <af/dim4.hpp>
+#include <af/defines.h>
+#include <af/traits.hpp>
+#include <vector>
+#include <iostream>
+#include <complex>
+#include <string>
+#include <testHelpers.hpp>
+
+using std::vector;
+using std::string;
+using std::cout;
+using std::endl;
+using af::cfloat;
+using af::cdouble;
+
+///////////////////////////////// CPP ////////////////////////////////////
+//
+// TEST(Cholesky, CPP)
```

Comment:
is this commented out unit test done on purpose ? Can a `TO DO` or `FIX ME` be added so that it will be un-commented or removed later.